docs/examples/patterns/contextual-menu/dark.html

```diff
@@ -65,71 +65,6 @@
 </span> vel eu.</p>
 
 <script>
-/**
-  Toggles the necessary aria- attributes' values on the menus
-  and handles to show or hide them.
-  @param {HTMLElement} element The menu link or button.
-  @param {Boolean} show Whether to show or hide the menu.
-*/
-function toggleMenu(element, show) {
-  var target = document.getElementById(element.getAttribute('aria-controls'));
-
-  if (target) {
-    element.setAttribute('aria-expanded', show);
-    target.setAttribute('aria-hidden', !show);
-  }
-}
-
-/**
-  Attaches event listeners for the menu toggle open and close click events.
-  @param {HTMLElement} menuToggle The menu container element.
-*/
-function setupContextualMenu(menuToggle) {
-  menuToggle.addEventListener('click', function(event) {
-    event.preventDefault();
-    var menuAlreadyOpen = menuToggle.getAttribute('aria-expanded') === 'true';
-    toggleMenu(menuToggle, !menuAlreadyOpen);
-  });
-}
-
-/**
-  Attaches event listeners for all the menu toggles in the document and
-  listeners to handle close when clicking outside the menu or using ESC key.
-  @param {String} contextualMenuToggleSelector The CSS selector matching menu toggle elements.
-*/
-function setupAllContextualMenus(contextualMenuToggleSelector) {
-  // Setup all menu toggles on the page.
-  var toggles = document.querySelectorAll(contextualMenuToggleSelector);
-
-  for (var i = 0, l = toggles.length; i < l; i++) {
-    setupContextualMenu(toggles[i]);
-  }
-
-  // Add handler for clicking outside the menu.
-  document.addEventListener('click', function(event) {
-    for (var i = 0, l = toggles.length; i < l; i++) {
-      var toggle = toggles[i];
-      var contextualMenu = document.getElementById(toggle.getAttribute('aria-controls'));
-      var clickOutside = !(toggle.contains(event.target) || contextualMenu.contains(event.target));
-
-      if (clickOutside) {
-        toggleMenu(toggle, false);
-      }
-    }
-  });
-
-  // Add handler for closing menus using ESC key.
-  document.addEventListener('keydown', function(e) {
-    e = e || window.event;
-
-    if (e.keyCode === 27) {
-      for (var i = 0, l = toggles.length; i < l; i++) {
-        toggleMenu(toggles[i], false);
-      }
-    }
-  });
-}
-
-setupAllContextualMenus('.p-contextual-menu__toggle');
+  {% include 'examples/patterns/contextual-menu/_script.js' %}
```

Comment:
Will this work for the JS tab in codepen include in docs?

Reply:
Yes, it does:

<img width="951" alt="Screenshot 2020-02-13 at 12 04 04" src="https://user-images.githubusercontent.com/83575/74428299-06b80300-4e59-11ea-8700-d7215c77bc0c.png">

CodePen (similar to example-js before) uses rendered example HTML (fetched via ajax), doesn't matter how it's done in the example template.